hyperspy/misc/example_signals_loading.py

```diff
@@ -45,3 +45,39 @@ def load_1D_EDS_TEM_spectrum():
     file_path = os.sep.join([os.path.dirname(__file__), 'eds',
                              'example_signals', '1D_EDS_TEM_Spectrum.hdf5'])
     return load(file_path)
+
+
+def load_object_hologram():
+    """
+    Load an object hologram image
+
```

Comment:
This seems to be a NOTES sections (we follow [numpy's style](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).